charter.md

```diff
@@ -2,92 +2,82 @@
-A "digital presentation" is a "W3C Verifiable Presentation" when it secures a JSON claimset that conforms to the W3C Technical Recommendation.
+The wording "Verifiable Credential" (VC) is used in VCDM 2.0. It implies the use of a schema defined using JSON-LD.
+The wording "Verifiable Presentation" (VP) is used in VCDM 2.0. It implies the use of data serialization using JSON-LD.
 
-A "digital credential" is an "OAuth Verifiable Credential" when it secures a JSON claimset that conforms to the requirements of documents developed by the OAuth WG. 
-A "digital presentation" is an "OAuth Verifiable Presentation" when it secures a JSON claimset that conforms to the requirements of documents developed by the OAuth WG. 
+In this charter, the VC concept is captured using the wording "digital credential", while the VP concept is captured using the wording "digital presentation".
 
-A "digital credential" might be a "mDoc verifiable credential", in other specifications.
-A "digital presentation" might be a "mDoc verifiable presentation", in other specifications.
+Some sets of claim names have been defined by the IANA and originate both from the IETF and the OIDF.
+The W3C has taken a different approach allowing to define any kind of claim names by referencing URNs using LD-JSON schemas that include the @context property.
 
-As a result, the SPICE WG avoids using the terms "verifiable credential" and "verifiable presentation" in this charter text, as it is not intended to draw exclusive association to these existing serializations.
+IETF and IRTF working groups have developed foundational building blocks with BBS Signatures, RSA Blind Signatures, Verifiable Random Functions,
+or other Selective-Disclosure and collection limitation techniques.
 
-## Key Design Properties of Digital Credentials & Presentations
+## Key Design Properties of Digital Credentials
 
-- Unlinkability (preventing tracking while proving attributes / capabilities)
-- Selective Disclosure (supporting user agency, data minimization, redaction)
-- Authenticity (ensuring information is trustworthy and can be authenticated)
+- Request and issuance of digital credentials
+- Request and renewal of digital credentials
+- Revocation request and checking of the revocation state of a digital credential
+- Unlinkability of digital presentations (preventing tracking between verifiers while proving attributes / capabilities)
+- Selective Disclosure of claims (supporting collection limitation and attribute redaction)
+- Zero Knowledge Proofs (demonstrating some properties about some attributes without disclosing their values)
+- Proof of ownership of a digital presentation (using a TA running in a TEE)
+- Detection or prevention of holders collaborative attacks
+- Authenticity (ensuring that the issuer of a digital credential can be either individually authenticated or authenticated as a member of a set of trusted issuers)
```

Comment:
```suggestion
- Unlinkability (preventing tracking while proving attributes / capabilities)
- Selective Disclosure (supporting user agency, data minimization, redaction)
- Authenticity (ensuring information is trustworthy and can be authenticated)
- Integrity (ensuring tampering with information is always evident)
- Confidentiality (ensuring information is protected from unauthorized access)
- Hardware Assurance (awareness of the security capabilities of issuers, holders and verifiers)
- Availability (efficiently information processing, minimizing data in flight, reducing carbon cost for storage and transmission, and supporting offline / paper credential use cases)
- Semantic interchangeability (ensuring terms are understood and used consistently, in a global or local context)
- Usability and Feasibility (the core operations must be easy to understand and use, and possible to integrate with existing platforms)
- Accountability (what format and protocol considerations can help mitigate abusive or excessive requests for credential presentation?  What mitigates abuse by issuers?)
```

Reply:
Agreed - let's keep the original framing

Reply:
The additions make clear the topics that will need to be addressed